graphbolt/src/csc_sampling_graph.cc

```diff
@@ -97,6 +97,48 @@ void CSCSamplingGraph::Save(torch::serialize::OutputArchive& archive) const {
   }
 }
 
+void CSCSamplingGraph::SetState(
+    torch::Dict<std::string, torch::Dict<std::string, torch::Tensor>>& state) {
+  auto&& regular_tensors = state.at("regular_tensors");
+  TORCH_CHECK(
+      regular_tensors.at("magic_number").equal(torch::tensor({1})),
```

Comment:
Add `static const int kPickleVersion = 6199` at L24 then:
```suggestion
      regular_tensors.at("version").equal(torch::tensor({kPickleVersion})),
```

Reply:
Update GetState accordingly.